packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -44,13 +44,14 @@ $mdc-toolbar-mobile-breakpoint: 599px;
     display: flex;
```

Comment:
Yes. AFAIK they are only necessary for toolbars with tabs inside of them, so this wouldn't have been fixed anywhere else.